.gitignore

```diff
@@ -7,3 +7,5 @@
 /laminas-mkdoc-theme/
 /phpunit.xml
 /vendor/
+.idea
```

Comment:
Please revert this and use your global Git configuration to ignore this folder. Otherwise we have to list all IDE-specific folders here.